client/state/login/magic-login/actions.js

```diff
@@ -72,66 +66,48 @@ export const fetchMagicLoginRequestEmail = email => dispatch => {
 			type: MAGIC_LOGIN_SHOW_CHECK_YOUR_EMAIL_PAGE,
 			email
 		} );
-	} ).catch( error => {
+	} ).catch( ( error ) => {
 		dispatch( {
 			type: MAGIC_LOGIN_REQUEST_LOGIN_EMAIL_ERROR,
 			error: error.message,
 		} );
 	} );
 };
 
-const authError = error => {
-	return {
-		type: MAGIC_LOGIN_REQUEST_AUTH_ERROR,
-		error,
-	};
-};
-
-const authSuccess = () => {
-	return {
-		type: MAGIC_LOGIN_REQUEST_AUTH_SUCCESS,
-	};
-};
-
+// @TODO should this move to `/state/data-layer`..?
```

Comment:
Reading the [readme file](https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/README.md) it looks like it's where we should put this type of code now. It's probably better if we move all login actions in a different pull request though.